CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 9.0.1
```

Comment:
This will also need a version bump in the pubspec :)

Reply:
![](https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExdDQ1enU0c2V0cDc2MDJyejI5a2RwYnV2Y3o5MXU2aG1ieHc3d3pjciZlcD12MV9naWZzX3NlYXJjaCZjdD1n/jUwpNzg9IcyrK/200.webp)